src/data/communityPageData.json

```
@@ -135,7 +135,7 @@
           "associatedLink": "/bsg-c000031"
         },
         {
-          "image": "assets/Community/LogoImageSingle/willy.jpg",
+          "image": "assets/Community/willy.jpg",
```

```suggestion
          "image": "assets/Community/wiley.jpg",
```

The image will also need to be renamed to support this.